### DISCUSSION TITLE
How to reduce execution time, sometimes it takes over 30 seconds to perform lighthouse audits.

### DISCUSSION THREAD

**Discussion by changkuan-jj** [GoogleChrome/lighthouse#15769] [Q&A]
I use Lighthouse Node CLI to run performance and accessibility audits. However, it takes more than 30s to finish . I want to know whether I can set some config to reduce the time. Thanks a lot.

**Comment by adamraine** [comment]
Using a [config](https://github.com/GoogleChrome/lighthouse/blob/main/docs/configuration.md) like this should help:

```js
const config = {
  extends: 'lighthouse:default',
  settings: {
    // Page load will time out after 10s instead of 30s
    maxWaitForLoad: 10_000,
  },
}
```
